backend/routes/userData.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+const userDataRouter = express.Router()
+
+userDataRouter.get('/', (req, res) => {
+  console.log('hitting user-data')
+  console.log(req.user)
+  if (req.user !== null && req.user !== undefined) {
```

Comment:
nit: this can just be `if (req.user) {`

Reply:
I'm doing it that way cause of the eslint strict boolean expression rule.
![image](https://user-images.githubusercontent.com/106122976/232856128-90e69614-28a1-40da-8470-1b48e228db7f.png)

Reply:
gotcha - explicitness isn't a bad thing, worth keeping it the way you have it then